src/NUnitFramework/framework/Internal/TestFilter.cs

```diff
@@ -208,7 +208,7 @@ private static TestFilter[] GetChildNodeFilters(TNode node)
         /// returns true when called. It never matches explicitly.
         /// </summary>
         [Serializable]
-        private class EmptyFilter : TestFilter
+        private sealed class EmptyFilter : TestFilter
```

Comment:
always seal classes when possible, maybe someone more familiar with dependencies should investigate what could be sealed or even made `internal`